test/e2e/updaterun_test.go

```diff
@@ -481,6 +496,18 @@ var _ = Describe("test CRP rollout with staged update run", func() {
 	})
 })
 
+func checkIfPlacedWorkResourcesOnAllMemberClusters() {
+	checkIfPlacedWorkResourcesOnMemberClustersInUpdateRun(allMemberClusters)
+}
+
+func checkIfPlacedWorkResourcesOnMemberClustersInUpdateRun(clusters []*framework.Cluster) {
```

Comment:
Why do we make this function specific to UpdateRun and put it here instead of utils_test.go?

Reply:
AFAIK, this function is only used by the updateRun especially that the time out used is update run specific.